src/main/java/io/confluent/connect/elasticsearch/ClientConfigCallbackHandler.java

```diff
@@ -210,4 +308,61 @@ private SSLContext sslContext() {
       }
     }
   }
+
+  private LoginContext login() throws PrivilegedActionException {
```

Comment:
Comment to describe this gets the context to log into ES via Kerberos/ or more descriptive method name will be helpful.